content/events/2019-02-02-project-hub.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "LP x CSSS: Project Hub ft. Intro to Machine Learning"
+date: 2018-02-02
+start_date: 2019-02-02T15:30:00.000Z
+end_date: 2019-02-02T18:30:00.000Z
+categories: [Event]
+tags: [Project Hub, Career, Networking, Workshop]
+aliases:
```

Comment:
We don't need to use aliases anymore, they are only needed for posts on the old site